assets/js/quizgame.js

```diff
@@ -0,0 +1,287 @@
+  currentQuestion.answers.forEach(renderAnswers);
+};
+
+// Once the answer has been selected, validate
+const answerValidation = function (event) {
+  const currentTarget = event.currentTarget;
+  const target = event.target;
+
+  const correctAnswer = currentTarget.getAttribute("data-answer");
+  const userAnswer = target.getAttribute("data-option");
+
+  // Initial IF statement to ensure the game isn't over, Second IF to carry out game logic
+  if (currentIndex < myQuestions.length - 1) {
+    if (correctAnswer === userAnswer) {
+      document.getElementById("question-element").remove();
+      currentIndex++;
+      renderQuestion();
+      countdownClock += 5;
+    } else {
+      document.getElementById("question-element").remove();
+      currentIndex++;
+      countdownClock -= 5;
+      renderQuestion();
+    }
+  } else {
+    renderScore();
+  }
+};
+
+// Start Quiz - remove elements, start timer and render question
+const startQuiz = function () {
+  startQuizDiv.remove();
+
+  renderTimer();
+
+  startTimer();
+
+  renderQuestion();
+};
+
+// Render Score: Remove last question, construct score container and allow user to submit initials
+const renderScore = function () {
+  document.querySelector("#clock").remove();
+  document.getElementById("question-element").remove();
+  const finalScore = countdownClock;
+
+  const scoreDiv = document.createElement("div");
+  scoreDiv.setAttribute("class", "score-div");
+  scoreDiv.setAttribute("id", "score-div");
+
+  const scoreContainer = document.createElement("div");
+  scoreContainer.setAttribute("id", "score-container");
+  scoreContainer.setAttribute("class", "score-container");
+
+  const scoreHeader = document.createElement("h2");
+  scoreHeader.setAttribute("id", "score-title");
+  scoreHeader.textContent = `Your Score Is: ${countdownClock}`;
+
+  const enterInitials = document.createElement("p");
+  enterInitials.setAttribute("class", "user-initials");
+  enterInitials.textContent = `Enter Initials:`;
+
+  const submitButton = document.createElement("button");
+  submitButton.setAttribute("id", "submit-btn");
+  submitButton.setAttribute("class", "submit-btn");
+  submitButton.textContent = `Submit Score`;
+
+  const userInitials = document.createElement("input");
+  userInitials.setAttribute("id", "user-input");
+  userInitials.setAttribute("class", "user-input");
+
+  scoreContainer.appendChild(scoreDiv);
+  scoreContainer.appendChild(scoreHeader);
+  scoreContainer.appendChild(enterInitials);
+  scoreContainer.appendChild(userInitials);
+  scoreContainer.appendChild(submitButton);
+  scoreContainer.addEventListener("click", answerValidation);
+
+  mainElement.appendChild(scoreContainer);
+
+  // Save Data to Local Storage
+  const saveData = function () {
+    const dataFromLS = localStorage.getItem("user-input");
+
+    const input = document.getElementById("user-input");
+    const inputArray = [];
+    let getUserCredentials = {
```

Comment:
Also - with objects we do not need to declare as `let`. We can still use `const` as the variable name is always pointing to the same object, regardless if we change what is in it. The same with arrays, we always use `const`. Think of the array and the object as boxes - even if we change the contents, the box is still the same on the outside.